src/shared/remoteSession.ts

```diff
@@ -5,9 +5,14 @@
 
 import * as vscode from 'vscode'
 import { Settings } from '../shared/settings'
+import { showMessageWithCancel } from './utilities/messages'
+import { Timeout } from './utilities/timeoutUtils'
 
 export async function openRemoteTerminal(options: vscode.TerminalOptions, onClose: () => void) {
-    await withoutShellIntegration(() => {
+    const timeout = new Timeout(60000)
+
+    await showMessageWithCancel('AWS: Starting session...', timeout)
```

Comment:
Could use the `showAfterMs` param to skip this message if the connection is fast enough.
```suggestion
    await showMessageWithCancel('AWS: Starting session...', timeout, 1000)
```